modules/runner/package.json

```diff
@@ -13,7 +13,6 @@
     "@ephox/bedrock-common": "^9.3.2",
     "@ephox/wrap-promise-polyfill": "^2.2.0",
     "jquery": "^3.4.1",
-    "mocha": "^8.2.1",
```

Comment:
Don't we still need this, just as a dev dependency? It looks like we still use mocha in the tests, it's just working due to this being listed as a dependency elsewhere in the repo.